### PR TITLE
Add Tests for ImageTextDectector class

### PR DESCRIPTION
**Pull request by saakshisaxena**
Tests for ImageTextDectector's `extractShoppingList` method, using Fake implementation ([pr #44](https://github.com/googleinterns/step256-2020/pull/44)) of TextDetectionAPI.

### CODE REVIEW

**Review by ralucatudor — APPROVED** [review]

File: src/test/java/com/google/sps/ImageTextDectectorTest.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    String expectedMessage = exceptionMessage;
+    String actualMessage = exception.getMessage();
+
+    Assertions.assertTrue(actualMessage.equals(expectedMessage));
+  }
+
+  /** Negative test for no text */
+  @Test
+  public void noText() throws Exception {
+    List<ShoppingListTextEntry> shoppingListText = new ArrayList<>();
```
Comment (ChangchengXiao) [inline src/test/java/com/google/sps/ImageTextDectectorTest.java:75]: Would it be better to call it "shoppingListTextEntries"?

File: src/test/java/com/google/sps/ImageTextDectectorTest.java [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    Exception exception =
+        Assertions.assertThrows(
+            PhotoDetectionException.class,
+            () -> {
+              imageTextDectector.extractShoppingList(IMAGE_BYTES);
+            });
+
+    String expectedMessage = "Shopping List doesn't contain any text";
+    String actualMessage = exception.getMessage();
+
```
Comment (ChangchengXiao) [inline src/test/java/com/google/sps/ImageTextDectectorTest.java:88]: No need to have this empty line.

**Review by ChangchengXiao — APPROVED** [review]

File: src/test/java/com/google/sps/ImageTextDectectorTest.java [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    initImageTextDectector(shoppingListTextEntries);
+
+    String expectedShoppingQuery = "Bag";
+    String actualShoppingQuery = imageTextDectector.extractShoppingList(IMAGE_BYTES);
+    Assert.assertEquals(expectedShoppingQuery, actualShoppingQuery);
+  }
+
+  @Test
+  public void multiWord_inSingleLine_Image() throws Exception {
```
Comment (skopecki) [inline src/test/java/com/google/sps/ImageTextDectectorTest.java:103]: Don't mix camelCase with underscores. I doubt that this is allowed by the style guide. Same below.

**Review by skopecki — APPROVED** [review]

File: src/test/java/com/google/sps/ImageTextDectectorTest.java (hunk 3 above)
Comment (saakshisaxena) [reply src/test/java/com/google/sps/ImageTextDectectorTest.java:103]: Oh yes! I was just thinking that it'll make it more readable.
Will change it.

**Pull request merged**